1-org/envs/shared/variables.tf

```diff
@@ -115,32 +115,41 @@ variable "project_budget" {
   budget_amount: The amount to use as the budget.
   alert_spent_percents: A list of percentages of the budget to alert on when threshold is exceeded.
   alert_pubsub_topic: The name of the Cloud Pub/Sub topic where budget related messages will be published, in the form of `projects/{project_id}/topics/{topic_id}`.
+  alert_spend_basis: The type of basis used to determine if spend has passed the threshold.
```

Comment:
Can you mention that the default is `FORCASTED_SPEND` and what the other possible values are?

Reply:
Done! Let me know if the comment matches the expectation

Reply:
LGTM